util/inteltool/gpio_groups.c

```diff
@@ -166,6 +167,9 @@ const struct gpio_community *const *get_gpio_communities(struct pci_dev *const s
 		*community_count = ARRAY_SIZE(icelake_pch_h_communities);
 		*pad_stepping = 16;
 		return icelake_pch_h_communities;
+	case PCI_DEVICE_ID_INTEL_CELERON_PENTIUM_SILVER_LPC:
+		*community_count = ARRAY_SIZE(geminilake_communities);
```

Comment:
@miczyg1 Is the default value of `*pad_stepping = 8` correct? I'm not quite sure about that.

Reply:
`*pad_stepping = 16;` is the correct one IIRC. Check the interval between subsequent DW0 registers